### PR TITLE
Sort instances!

### PR DESCRIPTION
**Pull request by sudo-flix**
sort instances. 
[Related Discord Message](https://discord.com/channels/871713465100816424/871713465578979351/1213121106286878771)

### CODE REVIEW

File: content/0.instances.md [hunk 1]
```diff
@@ -20,22 +20,22 @@ The community maintains these trusted instances, meaning they are likely to be u
...
-* https://bmov.vercel.app by [TheScreechingBagel](https://github.com/TheScreechingBagel) 💾🌐
-* https://stream.thehairy.me by [thehairy](https://github.com/thehairy) 💾🌐
-* https://sudo-flix.lol by [itzCozi](https://gitlab.com/itzCozi) 💾
-* https://scootydooter.vercel.app by [Toon](https://github.com/Toon-arch) 💾🌐
-* https://movie-web.x via [Unstoppable Domains](https://unstoppabledomains.com) and [IPFS](https://ipfs.tech) 💾
-    - https://movie-web.x is only accessible using Brave, Opera or installing an [extension](https://unstoppabledomains.com/extension) to resolve unstoppable domains
+- [mw.lonelil.com](https://mw.lonelil.com) by [lonelil - Partner](https://github.com/lonelil) » 💾🌐
+- [watch.qtchaos.de](https://watch.qtchaos.de) by [chaos - Project Lead](https://github.com/qtchaos) » 💾
+- [bmov](https://bmov.vercel.app) by [TheScreechingBagel - Mod](https://github.com/TheScreechingBagel) » 💾🌐
+- [stream.thehairy.me](https://stream.thehairy.me) by [thehairy - Mod](https://github.com/thehairy) » 💾🌐
```
Comment (thehairy) [inline content/0.instances.md:26]: My names comes first when sorted by alphabet, so I should come first! MUHAHAHA!
```suggestion
- [stream.thehairy.me](https://stream.thehairy.me) by [thehairy - Mod](https://github.com/thehairy) » 💾🌐
- [bmov](https://bmov.vercel.app) by [TheScreechingBagel - Mod](https://github.com/TheScreechingBagel) » 💾🌐
```

**Review by thehairy — CHANGES_REQUESTED** [review]

File: content/0.instances.md (hunk 1 above)
Comment (binaryoverload) [reply content/0.instances.md:26]: Denied

**Review by binaryoverload — APPROVED** [review]

File: content/0.instances.md (hunk 1 above)
Comment (sudo-flix) [reply content/0.instances.md:26]: logically this makes sense

**Comment by sudo-flix** [comment]
yay

**Pull request merged**